bin/common.sh

```diff
@@ -92,6 +92,36 @@ if [ "$SAS_COMMON_SOURCED" = "" ]; then
        fi
     fi
 
+    ## Check for air gap deployment
+    AIRGAP_DEPLOYMENT=${AIRGAP_DEPLOYMENT:-false}
```

Comment:
The common.sh file is sourced by just about every script, even those not deploying things.  It might make sense to move the airgap-specific logic to its own include file and only source it into the deploy_*.sh files.   Give that some thought.

Reply:
I have created a separate bin/airgap-include.sh file that is sourced in the deployment scripts.